### PR TITLE
changelog.py: remove dot from the end of an item description

### PR DESCRIPTION
**Pull request by vlad20012**

### CODE REVIEW

File: changelog.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def collect_changelog(repo: Repository, milestone: Milestone):
         issue_text = re.sub(comment_pattern, "", issue.body).replace("\r\n", "\n")
         result = re.search(changelog_description_pattern, issue_text)
         if result is not None:
-            description = result.group("description").strip()
+            description = result.group("description").strip().strip(".")
```
Comment (vlad20012) [inline changelog.py:134]: This actually means "remove `.` from the start and the end of the string", but sho would start a description with a dot? :thinking:

Comment (ortem) [reply changelog.py:134]: You can use `rstrip` method to ignore the leading characters

Comment (Undin) [reply changelog.py:134]: @vlad20012 ping

Comment (vlad20012) [reply changelog.py]: Replaced to `rstrip`

**Review by Undin — APPROVED** [review]

**Pull request merged**